app/index.jade

```diff
@@ -42,8 +42,8 @@ body(ng-controller="AppCtrl" ui-view="body" ng-mousemove="resetInactivityTime()"
   //- This does not depend on Angular and will show as soon as the images are done loading.
   .blocket-loading
     div
-      img.full(src="img/blocket-full.png")
-      img(src="img/blocket-no-boosters.png")
+      img.full(src="img/blocket-full.png" alt="Loading")
```

Comment:
Wrong example I guess. Inside index.jade we can't use AngularJS magic, so this line should just be in English.